src/components/InventoryTable/InventoryList.js

```diff
@@ -43,7 +44,7 @@ const ContextInventoryList = ({ showHealth, onRefreshData, ignoreRefresh, ...pro
 
     return (<Grid gutter="sm" className="ins-inventory-list">
         <GridItem span={ 12 }>
-            <InventoryEntityTable { ...props } onRefreshData={onRefreshData} />
+            <InventoryEntityTable { ...props } variant={TableVariant.compact} onRefreshData={onRefreshData} />
```

Comment:
This might be better as a `variant` prop of the `InventoryTable` with the TableVariant.compact set as the default.
There might be cases where an application explicitly uses a specific variant and setting it here would overwrite that.